src/App.tsx

```diff
@@ -93,29 +101,6 @@ export const App: React.FC = () => {
         } else {
           earth_orbits_container.style.transform = scale;
         }
-
-      } else if (windowWidth <= 500) {
-        const loss = 500 - windowWidth;
-        const percentLoss = Math.round(loss / 5);
-        const scaleNum = 1 - percentLoss / 100;
-
-        // Set the scale in the redux store
-        dispatch(setEarthOrbitsScale(scaleNum));
-
-        const scale = "scale(" + scaleNum + ")";
-
-        // Check if the element already has been scaled
-        const existingTransform = earth_orbits_container.style.transform;
-        if (existingTransform && existingTransform.includes("scale(")) {
-          const updatedTransform = existingTransform.replace(
-            /scale\([^)]*\)\s*/g,
-            ""
-          );
-          earth_orbits_container.style.transform =
```

Comment:
`adjsutElementScale()` doesn't work well for the mobile view so I am removing it for screen widths < 500.